Cargo.toml

```
@@ -40,6 +40,7 @@ serde = "1.0"
 anyhow = "1.0.33"
 unicode-width = "0.1"
 textwrap = "0.12"
+which = "4.0.2"
```

This could be os-gated to linux only.
```suggestion

[target.'cfg(target_os = "linux")'.dependencies]
which = "4.0.2"
```